src/backend/controllers/dom.ts

```diff
@@ -41,6 +44,17 @@ export class DomController extends BaseController {
     this.model = [];
     this.adapter = adapter;
     this.channel = channel;
+
+    this.callToRenderTree = new Rx.Subject();
+    this.callToRenderTree
+      .debounce((x) => {
+        return Rx.Observable.timer(250);
```

Comment:
I would remove the curlies and the round brackets on the argument to keep it as a one liner